source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -527,6 +527,23 @@ void Filter::sendTrailers(ProcessorState& state, const Http::HeaderMap& trailers
   stats_.stream_msgs_sent_.inc();
 }
 
+void Filter::onNewTimeout(const uint32_t message_timeout_ms) {
+  // The new timeout has to be >=1ms and <= max_message_timeout configured in filter.
+  const uint32_t min_timeout_ms = 1;
+  const uint32_t max_timeout_ms = config_->maxMessageTimeout();
+  if (message_timeout_ms < min_timeout_ms || message_timeout_ms > max_timeout_ms) {
+    ENVOY_LOG(warn, "Ext_proc server new timeout setting is out of range. "
```

Comment:
I think we should provide a better option than logging here an silent failure. Probably bump some stream stat on this

Reply:
Yes, good idea, done!